catalog/aws-load-balancer-operator/bundle.yaml

```diff
@@ -167,10 +173,10 @@ properties:
       name: Red Hat Inc.
       url: https://redhat.com
 relatedImages:
-- image: gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0
+- image: openshift.io/aws-load-balancer-operator:latest
   name: ""
-- image: quay.io/aws-load-balancer-operator/aws-load-balancer-operator-bundle:v1.1.0
+- image: quay.io/aws-load-balancer-operator/aws-load-balancer-operator-bundle:v1.2.0
   name: ""
-- image: quay.io/aws-load-balancer-operator/aws-load-balancer-operator:latest
+- image: quay.io/openshift/origin-kube-rbac-proxy:latest
```

Comment:
Can you add something to the commit message that explains/documents the changes to image registries and tags?
- kube-rbac-proxy now being pulled from quay with latest instead of specific tag
- aws-load-balancer-operator now being pulled from openshift.io instead of quay

Reply:
Updated the commit message to explain kube-rbac-proxy and annotation changes. `aws-load-balancer-operator ` image is just a placeholded used by default in [Makefile](https://github.com/openshift/aws-load-balancer-operator/blob/9181f50b15b34775b2cf2907bd6ea08d4f04f4f1/Makefile#L41), it's coming from `bundle/` and [substituted during CI tests](https://github.com/openshift/release/blob/143ef39a603c4c5aaf0e57599a5747abef7126bc/ci-operator/config/openshift/aws-load-balancer-operator/openshift-aws-load-balancer-operator-main.yaml#L27-L29).